src/qt/coincontroldialog.cpp

```diff
@@ -548,13 +548,10 @@ void CoinControlDialog::updateLabels(WalletModel *model, QDialog* dialog)
                 CTxOut txout(nChange, (CScript)std::vector<unsigned char>(24, 0));
                 if (txout.IsDust(dustRelayFee))
                 {
-                    if (CoinControlDialog::fSubtractFeeFromAmount) // dust-change will be raised until no dust
-                        nChange = txout.GetDustThreshold(dustRelayFee);
-                    else
-                    {
-                        nPayFee += nChange;
-                        nChange = 0;
-                    }
+                    nPayFee += nChange;
+                    nChange = 0;
+                    if (CoinControlDialog::fSubtractFeeFromAmount)
+                        nBytes -= 34; // we didn't detect lack of change above
```

Comment:
There are three instances where `nBytes -= 34` is occurs in this function now. I wonder if they could be unified, maybe by deleting the `&& !CoinControlDialog::fSubtractFeeFromAmount` condition on the third instance, and then getting rid of the first two instances?